frame/collective/src/tests.rs

```diff
@@ -89,7 +89,7 @@ pub type MaxMembers = ConstU32<100>;
 
 parameter_types! {
 	pub const MotionDuration: u64 = 3;
-	pub const MaxProposals: u32 = 100;
+	pub const MaxProposals: u32 = 512;
```

Comment:
Please add a small test that fails if `MaxProposals` is larger than 2^16, since we know that the benchmarks would not work with that.

Reply:
It shouldn't fail at 2^16, 2^32 should be the value when it fails.

Reply:
We could even restrict `MaxPropsoal` to `u16` here, or increase the length everywhere so that we can use `u32`(Tbh I don't see the point of that since that would really slow down the benchmarks)

Reply:
We normally use `u32` as convention, even though the benchmark will most likely never work with 2³².  
Using 4 byte would increase the base weight of the benchmarking result, so its better to keep the lower bounds low.